packages/storybook-react/src/stories/OrderedList/OrderedListItem.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { OrderedList, OrderedListItem } from '@utrecht/component-library-react';
+import React from 'react';
+import { itemArgTypes } from './argTypes';
+
+export default {
+  title: 'React.js Component/Ordered List/Item',
+  id: 'react-ordered-list-item',
+  component: OrderedListItem,
+  decorators: [(Story) => <OrderedList>{Story()}</OrderedList>],
+  argTypes: itemArgTypes,
+} as ComponentMeta<typeof OrderedListItem>;
+
+export const Default: ComponentStory<typeof OrderedListItem> = (args) => <OrderedListItem {...args} />;
+
+Default.args = {
+  children: 'List item',
+};
+
+// Configure a `displayName` for the code snippet when used as Storybook subcomponent
+Default.displayName = 'OrderedListItem';
```

Comment:
TypeScript complains about this locally 
```ts
Property 'displayName' does not exist on type 'ComponentStory<ForwardRefExoticComponent<OrderedListItemProps & RefAttributes<HTMLLIElement>>>'.ts(2339)
```

Also, do you think we need it?
Because we already set the  `displayName` in the [React component implementation](https://github.com/nl-design-system/utrecht/blob/main/packages/component-library-react/src/OrderedListItem.tsx#L22)

Reply:
The `displayName` is for the Storybook Template component that wraps our component that indeed already has the display name.

I added it to fix this issue: https://github.com/storybookjs/storybook/issues/12596#issuecomment-723440097